### PR TITLE
Removing sed subprocess call when attempting to take screenshot. Unne…

### LINKED ISSUE #22: Taking screenshots produces empty file

### PR DESCRIPTION
**Pull request by gturbyne**
…cessary since writing directly to binary file

Fixes #22

### CODE REVIEW

File: orchestrator/src/androidtestorchestrator/device.py [hunk 1]
```diff
@@ -527,15 +527,9 @@ def take_screenshot(self, local_screenshot_path: str) -> None:
...
-        # Remember Android is always unix-like paths (aka do not use os.path.join):
-        base_name = os.path.basename(local_screenshot_path)
-        device_path = f"{self.external_storage_location}/{base_name}"
         with open(local_screenshot_path, 'w+b') as f:
-            # according to https://blog.shvetsov.com/2013/02/grab-android-screenshot-to-computer-via.html, need
-            # to use sed to remove unwanted carriage return/linefeeds
-            p = subprocess.Popen(["sed", "'s/\r$//'"], stdin=subprocess.PIPE, stdout=f)
-            self.execute_remote_cmd("shell", "screencap", "-p", device_path, capture_stdout=False,
-                                    stdout_redirect=p.stdin,
+            self.execute_remote_cmd("shell", "screencap", "-p", capture_stdout=False,
```
[the remark this answers is not in the document]
Comment (li-wjohnson) [reply orchestrator/src/androidtestorchestrator/device.py]: Or rather, have you tried opening the screenshot after this change?

[the remark this answers is not in the document]
Comment (gturbyne) [reply orchestrator/src/androidtestorchestrator/device.py]: Yes, I was able to open multiple screenshots after this change.

[the remark this answers is not in the document]
Comment (li-wjohnson) [reply orchestrator/src/androidtestorchestrator/device.py]: I'm confused. adb shell does some stuff with line-endings... I never got adb shell screencap to work locally. I have to use adb exec-out screencap or that sed trick. Do you know why it works?

[the remark this answers is not in the document]
Comment (li-wjohnson) [reply orchestrator/src/androidtestorchestrator/device.py]: See here: https://stackoverflow.com/questions/13578416/read-binary-stdout-data-from-adb-shell

[the remark this answers is not in the document]
Comment (gturbyne) [reply orchestrator/src/androidtestorchestrator/device.py]: Ya, I saw similar (if not the same) posts about this same issue. It seems as though Google might have fixed some of the discrepancies between shell and exec-out. I could look through their changes and see if I can find anything recent.  

Looking through some of the example outputs from the same link you posted above, I'm not seeing some of the issues that people originally reported there, for example:

$ adb shell "echo -n '\x0a'" | xxd -g1
00000000: 0a 
(This use to output 00000000: 0D 0A)
$ adb exec-out "echo -n '\x0a'" | xxd -g1
00000000: 0a

My adb version is 28.0.3 by the way

[the remark this answers is not in the document]
Comment (li-wjohnson) [reply orchestrator/src/androidtestorchestrator/device.py]: Well, that's super weird. Ship it!

**Pull request merged**